src/test/kotlin/org/prebid/cache/functional/testcontainers/container/AerospikeContainer.kt

```diff
@@ -0,0 +1,23 @@
+package org.prebid.cache.functional.testcontainers.container
+
+import org.testcontainers.containers.GenericContainer
+
+class AerospikeContainer(imageName: String) : GenericContainer<AerospikeContainer>(imageName) {
+
+    companion object {
+        const val PORT = 3000
+        const val NAMESPACE = "prebid_cache"
+    }
```

Comment:
According to official Kotlin style guide(https://kotlinlang.org/docs/coding-conventions.html#class-layout) companion object should be declared at the very bottom of class file.